themes-default/slim/src/components/snatch-selection.vue

```diff
@@ -89,15 +72,28 @@ export default {
             $('#summaryBackground').height(height);
             $('#summaryBackground').offset({ top, left: 0 });
             $('#summaryBackground').show();
-        },
-        reverse(array) {
-            return array ? array.slice().reverse() : [];
-        },
-        dedupeGenres(genres) {
-            return genres ? [...new Set(genres.slice(0).map(genre => genre.replace('-', ' ')))] : [];
         }
     },
     mounted() {
+        const {
+            indexer,
+            id,
+            show,
+            getShow,
+            $store
+        } = this;
+
+        // Let's tell the store which show we currently want as current.
+        $store.commit('currentShow', {
+            indexer,
+            id
+        });
+
+        // We need the show info, so let's get it.
+        if (!show) {
+            getShow({ id, indexer, detailed: false });
```

Comment:
showQueueStatus could be moved to a subroute. Like: `api/v2/series/tvdb114801/operation/queuestatus`. And maybe use websockets at a later stage. But i'm honestly at the moment not sure about that.

Size should get size stored from db. Then it could be moved under the non-detailed part.